components/content/Img/Img.ts

```diff
@@ -1,50 +1,56 @@
-import { LitElement, css, html } from 'lit';
+import { LitElement, html } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
 import { JSXCustomElement } from '../../../types/jsx-custom-element';
 
+export interface ImgAttributes {
+  block?: boolean;
+  responsive?: boolean;
+  aspectRatio?: string;
+  radius?: boolean;
+  objectFit?: 'cover' | 'contain';
+  objectPosition?: string;
+}
+
 @customElement('diamond-img')
 export class Img extends LitElement {
-  @property({ reflect: true }) readonly fit?: 'contain' | 'cover' | 'fill';
-
-  static styles = css`
-    :host {
-      display: block;
-    }
-
-    ::slotted(img) {
-      width: 100%;
-      height: auto;
-    }
-
-    ::slotted(img[fit='contain']) {
-      object-fit: contain;
-    }
-
-    ::slotted(img[fit='cover']) {
-      object-fit: cover;
-    }
-
-    ::slotted(img[fit='fill']) {
-      object-fit: fill;
-    }
-  `;
+  @property({ type: Boolean, reflect: true }) block = false;
+  @property({ type: Boolean, reflect: true }) responsive = false;
+  @property({ reflect: true, attribute: 'aspect-ratio' }) aspectRatio: string =
+    'auto';
+  @property({ type: Boolean, reflect: true }) radius = false;
+  @property({ reflect: true, attribute: 'object-fit' }) objectFit:
+    | 'fill'
+    | 'cover'
+    | 'contain' = 'fill';
+  @property({ reflect: true, attribute: 'object-position' })
+  objectPosition: string = '50% 50%';
```

Comment:
```suggestion
  objectPosition = '50% 50%';
```
TS will know it's a string